### PR TITLE
Create pull_request_template.md

### LINKED ISSUE #33: [Doc] Create a pull request template
We're lacking a PR template, we should create one

### PR DESCRIPTION
**Pull request by rheone**
This contribution updates documentation. It adds a pull request template.

Closes #33

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Hello, and welcome! Thanks you for your contribution to **Gulliver**.
+
+1. Please let us know about the contribution you are making
+   - [ ] Does this contribution update documentation?
+   - [ ] Does this contribution update code?
+      - [ ] Does it fix a bug?
+      - [ ] Does it add new functionality?      
+
+1. Assuming there is a code change in this contribution
+   - [ ] Have you included unit test to verify the change?
```
Comment (ahsteele) [inline .github/pull_request_template.md]: unit test(s)

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+   - [ ] Do all pre-existing unit tests pass as expected?
+   - [ ] Do the change break existing functionality, if so have you explained why is such appropriate?
```
Comment (ahsteele) [inline .github/pull_request_template.md]: Does the change break existing functionality, if so have you explained why doing so is necessary?

**Comment by rheone** [comment]
corrected

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Hello, and welcome! Thanks you for your contribution to **Arcus**.
```
Comment (ahsteele) [inline .github/pull_request_template.md]:
```suggestion
Hello, and welcome! Thanks you for your contribution to **Gulliver**.
```

Comment (rheone) [reply .github/pull_request_template.md]: Stupid copy and paste

**Pull request merged**